starknet_py/net/account/account_client.py

```diff
@@ -226,6 +229,7 @@ async def prepare_invoke_function(
         :param version: Transaction version
         :return: InvokeFunction created from the calls (without the signature)
         """
+        self._assert_version_matches_supported_tx_version(version)
```

Comment:
We use 0 as a default now.

Reply:
I don't get this comment. Could you explain?